ICAFIX/hcp_fix_multi_run

```diff
@@ -177,13 +191,47 @@ else
 	this_script_dir=$(dirname "$0")
 fi
 
-# All fix settings are held in the settings.sh file - edit this file to suit your setup
+# determine if legacy fix or pyfix will be used
+legacyFixEnabled=$(opts_StringToBool "$legacyFixEnabled")
+if [[ (($legacyFixEnabled)) && -f "${FSL_FIXDIR:-}/fix"  && ! -f "${FSL_FIXDIR:-}/convert_fix_model" ]];then
```

Comment:
If `legacyFixEnabled` is TRUE, but `FSL_FIXDIR` doesn't exist, I think that should be an error, because those two things are in conflict.

Reply:
The request was to either have the FSL_FIXDIR set or to specify it with the commandline option.  If that is not what it does, it needs to be revised.

Reply:
The idea is that "enabled" means it will look for it.  If the variable isn't set, or points somewhere that doesn't have what is expected, then it will use pyfix instead.

Reply:
If we had the option be just "use pyfix" or "use legacy fix", and error if it can't, then that makes it harder to select the behavior based on whether the user has an old `SetUp...` pointing to legacy fix.

Reply:
See my proposal above.  I think it is totally appropriate in this situation to generate an error if there is a conflict between an argument (calling for either `pyfix` or matlab-based `fix`) and their intended `SetUp`.  This is major change to how things are working "under the hood", so having to deal with an error once seems a good way to inform users of the change.